### PR TITLE
Unit tests realization process

### PR DESCRIPTION
**Pull request by mx-crs**
- unit tests for swagger2locustio/generators/base_generator.py module
- unit tests for swagger2locustio/strategy/json_strategy.py module

### CODE REVIEW

File: swagger2locustio/templates/locustfile_templates.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 from jinja2 import Template
 
 
-file_template = Template("""
-import os
+file_template = Template("""import os
```
Comment (DmitryPoliuha) [inline swagger2locustio/templates/locustfile_templates.py:4]: tabulation is very important in the case of generation locustfile. have you checked file generation after changing this line?

File: swagger2locustio/templates/locustfile_templates.py [hunk 2]
```diff
@@ -23,7 +22,7 @@ def on_start(self):
 class WebsiteUser(HttpLocust):
     task_set = Tests
     wait_time = between(5.0, 9.0)
-    {% if host %}
+{% if host %}
```
Comment (DmitryPoliuha) [inline swagger2locustio/templates/locustfile_templates.py:25]: tabulation is very important in the case of generation locustfile. have you checked file generation after changing this line?

File: swagger2locustio/generators/base_generator.py [hunk 3]
```diff
@@ -35,6 +35,8 @@ def generate_test_cases(self, paths_data: dict) -> str:
                 except ValueError as e:
                     logging.warning(e)
                     continue
+
+            # Issue: how about required params which I guess will not be used in some of these cases in such realization?
```
[the remark this answers is not in the document]
Comment (mx-crs) [reply swagger2locustio/generators/base_generator.py:39]: yep, sorry, I didn't see it in self._create_params_combinations

File: swagger2locustio/templates/locustfile_templates.py (hunk 1 above)
Comment (mx-crs) [reply swagger2locustio/templates/locustfile_templates.py:4]: yes, I've checked it

File: swagger2locustio/templates/locustfile_templates.py (hunk 2 above)
Comment (mx-crs) [reply swagger2locustio/templates/locustfile_templates.py:25]: yes, I've checked it

File: swagger2locustio/generators/base_generator.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (DmitryPoliuha) [reply swagger2locustio/generators/base_generator.py]: I think that this is my fail too so I need to choose better names for methods =)

**Pull request merged**